docs/walkthrough_transcribe.md

```diff
@@ -2,22 +2,112 @@
+
+![image: Transcription panel on eScriptorium (access button framed in red)](img/transcribe/transcription_panel.png "Transcription panel on eScriptorium (access button framed in red)")
+
+![image: Text panel on eScriptorium (access button framed in red)](img/transcribe/text_panel.png "Text panel on eScriptorium (access button framed in red)")
+
+### Transcription by hand
+
+When the transcription has to be done by hand, the best option is to use the "Transcription" panel.  
+
+To start transcribing, click on the corresponding zone. An input window is displayed. To record a transcription, press ++enter++: the transcription interface automatically displays the input field for the next segment.
+
+![image: Illustration of the transcription of the text by hand.](img/transcribe/by_hand.gif "Illustration of the transcription of the text by hand")
+
+<!-- I don't think this makes sense:
+!!! Note
+    If the transcription does not match the segmentation of the image, you have the possibility to modify the transcription in the lines through one of the two panels, until it matches completely.
+-->
+
 ## Automatic transcription
 
-## Virtual Keyboard
+You need a model to transcribe automatically, here are some ways to get one:  
+
+- by [training one with eScriptorium](walkthrough_train.md) or (via the command line) with [kraken](https://kraken.re),
+- by having one shared with you <!--either--> within [eScriptorium](walkthrough_collaborate.md#share-a-model),<!--or sent to you as a file through other means of communication,-->
+- by importing a model from a file. Such files can be retrieved on the [Zenodo community](https://zenodo.org/communities/ocr_models/) dedicated to Kraken models. Then can also be shared with you individually through other means of communication.
+
+### Predicting the transcription
+
+To learn how to apply the text recognition model and obtain a prediction, see the section ["Predict a segmentation or a transcription"](walkthrough_predict.md)
+
+### Editing the transcription
+
+If you want to edit the transcription, click on the "Edit" button, available at `{base_url}/document/{document-id}/images/` or in a square when you hover upon the thumbnail of an image. Make sure that you are on the right [transcription version](#transcription-versions) by checking the drop-down menu on the top-right corner of the page. Once it is done, you can click on the first line and then check, line by line, the transcription and correct it if needed. Press "Enter" to validate the change and go on to the next line.
+
+![image: Correction of the transcription.](img/transcribe/correction.gif "Correction of the transcription")
+
+!!! Tip
+    If you need to had special characters in your transcription and they are not available on your own keyboard, you can set and use a [virtual keyboard](walkthrough_virtual_keyboard.md).
+
+!!! Tip
```

Comment:
As discussed, it would probably make a lot of sense to have a separate subsection dedicated to "history"